<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GADGETS</class>
 <widget class="QWidget" name="GADGETS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GADGETS</string>
  </property>
  <widget class="QGroupBox" name="groupBoxUsb_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>320</width>
     <height>420</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButtonUsbIp">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
     </font>
    </property>
    <property name="text">
     <string>NetworkConfiguration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOk">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>274</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonnormal">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>224</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Normal Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonUSB">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>134</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Network Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSerial">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>94</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Serial Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSDCard">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>184</y>
      <width>109</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SD Card</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxUsbOnoff">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>14</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>USB </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUSBONOFF">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>9</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxUsb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>320</width>
     <height>420</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelText">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>40</y>
      <width>330</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DEVICE NETWORK GADGET</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHostIpaddress">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Host IPAddress</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUsb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHost">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>[F2] Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonusbExit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[F4] Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUsbIpaddress_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>USB IPAddress</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
